### DISCUSSION TITLE
筷筷小报#4 - 怎样提出一个好问题？

### DISCUSSION THREAD

**Discussion by giscus** [ztkuaikuai/blog#9] [General]
# 筷筷小报#4 - 怎样提出一个好问题？

记录一些技术、生活、阅读的感悟；是我的人类使用说明书。

https://blog.kuaikuaitz.top/posts/%E7%AD%B7%E7%AD%B7%E5%B0%8F%E6%8A%A54

**Comment by ztkuaikuai** [comment]
如果有人想蹲 Follow 邀请码，可以在这里评论~

**Reply by chttiu** [reply]
我想要呀

**Reply by ztkuaikuai** [reply]
今天刚好有一个，加我wx发你~kuaikuaitz

**Reply by chttiu** [reply]
大佬，加你了，同意一下

**Reply by chttiu** [reply]
没想到评论，能这么快被看到，惊喜
